apps/frontend/src/pages/analytics/libs/helpers/get-date-range/get-date-range.helper.ts

```diff
@@ -0,0 +1,21 @@
+import { formatDate } from "~/libs/helpers/helpers.js";
+
+const DAYS_TO_ADVANCE = 1;
+
+const getDateRange = (startDay: Date, endDay: Date): string[] => {
+	const dates: Date[] = [];
+	const currentDate = new Date(startDay);
+
+	while (currentDate <= endDay) {
+		dates.push(new Date(currentDate));
+		currentDate.setDate(currentDate.getDate() + DAYS_TO_ADVANCE);
```

Comment:
Not critical, but we have a library for dealing with dates (date-fns), I think you might get good use of that here.

Reply:
Agree